### PR TITLE
Removing Macros that were excluded

### PR DESCRIPTION
**Pull request by Sebanisu**
There were some macros that were excluded from https://github.com/codereport/jsource/pull/132. 
1)	I manually comment out the excluded Macro. From my exclude text file.
2)	I run my python script
3)	Commit change
4)	See what breaks.
5)	Nothing breaks
a.	At least 2 macros were okay for removal after revision of the python script.
b.	Some macros were revived in main branch after merging pull requests.
6)	The macro was used in another macro in one file.
a.	I add the macro back using #define above and #undef below the usage.
b.	At least one macro had more than one usage in only one file.
7)	The macro was used in multiple files and multiple macros.
a.	I put the macro back in the `ja.h` file.
b.     I tried to add a comment saying why the macro wasn't removed.
8)	In all cases at least the usage of these macros was reduced to only a handful.
9)	I found a few cases where macros were #undef and the next line #define pointing to a new function. I manually removed the few of those I saw.

based on: https://github.com/codereport/jsource/pull/132
related: https://github.com/codereport/jsource/issues/13

This log isn't really accurate. There was a lot of manual work done. After the python script ran. See above.

|Path|Macro Name|Function|Regex|Replace|
|---|---|---|---|---|
|`jsrc\ja.h`|#define bindd(x,y)                  jtbindd(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`||
|`jsrc\verbs\va2ss.c` `jsrc\verbs\vbang.c`|bindd|jtbindd|`(^\|[ \t]+\|[^\d\w_])bindd\((?=([^,]+?),([^)]+?)\))`|`\1jtbindd(jt,`|
|`jsrc\ja.h`|#define binzz(x,y)                  jtbinzz(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`||
|`jsrc\ja.h`|#define dgcd(x,y)                   jtdgcd(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`||
|`jsrc\k.c` `jsrc\verbs\va2ss.c` `jsrc\verbs\ve.c`|dgcd|jtdgcd|`(^\|[ \t]+\|[^\d\w_])dgcd\((?=([^,]+?),([^)]+?)\))`|`\1jtdgcd(jt,`|
|`jsrc\ja.h`|#define dlcm(x,y)                   jtdlcm(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\va2ss.c` `jsrc\verbs\ve.c`|dlcm|jtdlcm|`(^\|[ \t]+\|[^\d\w_])dlcm\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define equ(x,y)                    jtequ(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\k.c` `jsrc\sl.c` `jsrc\vq.h` `jsrc\adverbs\ai.c` `jsrc\adverbs\am.c` `jsrc\adverbs\am1.c` `jsrc\adverbs\amn.c` `jsrc\adverbs\ar.c` `jsrc\adverbs\as.c` `jsrc\conjunctions\ca.c` `jsrc\conjunctions\cc.c` `jsrc\conjunctions\cf.c` `jsrc\conjunctions\ch.c` `jsrc\conjunctions\cp.c` `jsrc\conjunctions\cpdtsp.c` `jsrc\conjunctions\crs.c` `jsrc\conjunctions\cu.c` `jsrc\conjunctions\cx.c` `jsrc\debugging\d.c` `jsrc\parsing\pv.c` `jsrc\representations\r.c` `jsrc\representations\rl.c` `jsrc\verbs\v.c` `jsrc\verbs\v0.c` `jsrc\verbs\v1.c` `jsrc\verbs\v2.c` `jsrc\verbs\va2s.c` `jsrc\verbs\vb.c` `jsrc\verbs\vcat.c` `jsrc\verbs\vd.c` `jsrc\verbs\vi.c` `jsrc\verbs\visp.c` `jsrc\verbs\vo.c` `jsrc\verbs\vp.c` `jsrc\verbs\vrep.c` `jsrc\verbs\vs.c` `jsrc\verbs\vsb.c` `jsrc\verbs\vx.c` `jsrc\xenos\xs.c`|equ|jtequ|`(^\|[ \t]+\|[^\d\w_])equ\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define fplus(x,y)                  jtfplus(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\v0.c`|fplus|jtfplus|`(^\|[ \t]+\|[^\d\w_])fplus\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define ftymes(x,y)                 jtftymes(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\v0.c`|ftymes|jtftymes|`(^\|[ \t]+\|[^\d\w_])ftymes\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define igcd(x,y)                   jtigcd(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\va2ss.c` `jsrc\verbs\ve.c`|igcd|jtigcd|`(^\|[ \t]+\|[^\d\w_])igcd\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define ilcm(x,y)                   jtilcm(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\va2ss.c`|ilcm|jtilcm|`(^\|[ \t]+\|[^\d\w_])ilcm\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define intpow(x,y)                 jtintpow(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\va2ss.c` `jsrc\verbs\vm.c`|intpow|jtintpow|`(^\|[ \t]+\|[^\d\w_])intpow\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h` `jsrc\ja.h`|#define mult(x,y)                   jtmult(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\dtoa.c` `jsrc\representations\rt.c` `jsrc\verbs\va2.c` `jsrc\verbs\vg.c` `jsrc\verbs\vi.c` `jsrc\verbs\vo.c` `jsrc\verbs\vrep.c` `jsrc\dtoa.c` `jsrc\representations\rt.c` `jsrc\verbs\va2.c` `jsrc\verbs\vg.c` `jsrc\verbs\vi.c` `jsrc\verbs\vo.c` `jsrc\verbs\vrep.c`|mult|jtmult|`(^\|[ \t]+\|[^\d\w_])mult\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h` `jsrc\ja.h`|#define over(x,y)                   jtover(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\sl.c` `jsrc\sn.c` `jsrc\adverbs\am1.c` `jsrc\adverbs\amn.c` `jsrc\adverbs\ao.c` `jsrc\adverbs\ap.c` `jsrc\adverbs\ar.c` `jsrc\adverbs\as.c` `jsrc\conjunctions\cc.c` `jsrc\conjunctions\cg.c` `jsrc\conjunctions\ch.c` `jsrc\conjunctions\cx.c` `jsrc\debugging\dc.c` `jsrc\format\f2.c` `jsrc\representations\rl.c` `jsrc\representations\rt.c` `jsrc\verbs\v0.c` `jsrc\verbs\v2.c` `jsrc\verbs\vcant.c` `jsrc\verbs\vcat.c` `jsrc\verbs\vd.c` `jsrc\verbs\ve.c` `jsrc\verbs\vf.c` `jsrc\verbs\vfromsp.c` `jsrc\verbs\vi.c` `jsrc\verbs\visp.c` `jsrc\verbs\vrep.c` `jsrc\verbs\vs.c` `jsrc\verbs\vt.c` `jsrc\verbs\vx.c` `jsrc\sl.c` `jsrc\sn.c` `jsrc\adverbs\am1.c` `jsrc\adverbs\amn.c` `jsrc\adverbs\ao.c` `jsrc\adverbs\ap.c` `jsrc\adverbs\ar.c` `jsrc\adverbs\as.c` `jsrc\conjunctions\cc.c` `jsrc\conjunctions\cg.c` `jsrc\conjunctions\ch.c` `jsrc\conjunctions\cx.c` `jsrc\debugging\dc.c` `jsrc\format\f2.c` `jsrc\representations\rl.c` `jsrc\representations\rt.c` `jsrc\verbs\v0.c` `jsrc\verbs\v2.c` `jsrc\verbs\vcant.c` `jsrc\verbs\vcat.c` `jsrc\verbs\vd.c` `jsrc\verbs\ve.c` `jsrc\verbs\vf.c` `jsrc\verbs\vfromsp.c` `jsrc\verbs\vi.c` `jsrc\verbs\visp.c` `jsrc\verbs\vrep.c` `jsrc\verbs\vs.c` `jsrc\verbs\vt.c` `jsrc\verbs\vx.c`|over|jtover|`(^\|[ \t]+\|[^\d\w_])over\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define pospow(x,y)                 jtpospow(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\va2ss.c` `jsrc\verbs\vm.c`|pospow|jtpospow|`(^\|[ \t]+\|[^\d\w_])pospow\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define qbin(x,y)                   jtqbin(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define qcompare(x,y)               jtqcompare(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define qdiv(x,y)                   jtqdiv(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\v0.c` `jsrc\verbs\vgauss.c` `jsrc\verbs\vq.c`|qdiv|jtqdiv|`(^\|[ \t]+\|[^\d\w_])qdiv\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define qgcd(x,y)                   jtqgcd(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vq.c`|qgcd|jtqgcd|`(^\|[ \t]+\|[^\d\w_])qgcd\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define qlcm(x,y)                   jtqlcm(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define qminus(x,y)                 jtqminus(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ap.c` `jsrc\verbs\v0.c` `jsrc\verbs\vgauss.c` `jsrc\verbs\vq.c` `jsrc\words\wn.c`|qminus|jtqminus|`(^\|[ \t]+\|[^\d\w_])qminus\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define qplus(x,y)                  jtqplus(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ao.c` `jsrc\adverbs\ap.c` `jsrc\verbs\v0.c` `jsrc\words\wn.c`|qplus|jtqplus|`(^\|[ \t]+\|[^\d\w_])qplus\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define cvz(x,y)                    jtcvz(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define daxis(x,y)                  jtdaxis(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define dbin(x,y)                   jtdbin(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define ebarvec(x,y)                jtebarvec(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define ifb(x,y)                    jtifb(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define pfill(x,y)                  jtpfill(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define pind(x,y)                   jtpind(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define poly2a(x,y)                 jtpoly2a(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define probelocal(x,y)             jtprobelocal(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define prod(x,y)                   jtprod(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define pscan(x,y)                  jtpscan(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define qpow(x,y)                   jtqpow(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define xd1(x,y)                    jtxd1(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define xexp(x,y)                   jtxexp(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define xrep(x,y)                   jtxrep(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define qq(x,y)                     jtqq(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ai.c` `jsrc\adverbs\ar.c` `jsrc\conjunctions\cc.c` `jsrc\conjunctions\cip.c` `jsrc\conjunctions\cpdtsp.c` `jsrc\conjunctions\cu.c` `jsrc\parsing\pv.c` `jsrc\verbs\vf.c` `jsrc\verbs\vm.c` `jsrc\xenos\xb.c` `jsrc\xenos\xfmt.c`|qq|jtqq|`(^\|[ \t]+\|[^\d\w_])qq\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define qrem(x,y)                   jtqrem(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define qtymes(x,y)                 jtqtymes(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ao.c` `jsrc\verbs\v0.c` `jsrc\verbs\vgauss.c` `jsrc\verbs\vq.c`|qtymes|jtqtymes|`(^\|[ \t]+\|[^\d\w_])qtymes\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define remdd(x,y)                  jtremdd(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\ve.c`|remdd|jtremdd|`(^\|[ \t]+\|[^\d\w_])remdd\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define remid(x,y)                  jtremid(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h` `jsrc\ja.h`|#define roll2(x,y)                  jtroll2(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vrand.c`|roll2|jtroll2|`(^\|[ \t]+\|[^\d\w_])roll2\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\verbs\vrand.c`|roll2|jtroll2|`(^\|[ \t]+\|[^\d\w_])roll2\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\verbs\vrand.c`|roll2|jtroll2dot|||
|`jsrc\ja.h`|#define rollany(x,y)                jtrollany(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vrand.c`|rollany|jtrollany|`(^\|[ \t]+\|[^\d\w_])rollany\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xcompare(x,y)               jtxcompare(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\k.c` `jsrc\vq.h` `jsrc\vx.h` `jsrc\adverbs\ao.c` `jsrc\format\f2.c` `jsrc\verbs\v0.c` `jsrc\verbs\v2.c` `jsrc\verbs\vgcomp.c` `jsrc\verbs\vq.c` `jsrc\verbs\vx.c`|xcompare|jtxcompare|`(^\|[ \t]+\|[^\d\w_])xcompare\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xgcd(x,y)                   jtxgcd(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vq.c` `jsrc\verbs\vx.c`|xgcd|jtxgcd|`(^\|[ \t]+\|[^\d\w_])xgcd\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xlcm(x,y)                   jtxlcm(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vq.c`|xlcm|jtxlcm|`(^\|[ \t]+\|[^\d\w_])xlcm\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xminus(x,y)                 jtxminus(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\k.c` `jsrc\adverbs\ap.c` `jsrc\verbs\v2.c` `jsrc\verbs\vq.c` `jsrc\verbs\vx.c`|xminus|jtxminus|`(^\|[ \t]+\|[^\d\w_])xminus\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xplus(x,y)                  jtxplus(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ao.c` `jsrc\adverbs\ap.c` `jsrc\format\f2.c` `jsrc\verbs\v0.c` `jsrc\verbs\vq.c` `jsrc\verbs\vx.c`|xplus|jtxplus|`(^\|[ \t]+\|[^\d\w_])xplus\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xpow(x,y)                   jtxpow(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\k.c` `jsrc\conjunctions\ca.c` `jsrc\format\f2.c` `jsrc\verbs\v2.c`|xpow|jtxpow|`(^\|[ \t]+\|[^\d\w_])xpow\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xrem(x,y)                   jtxrem(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\v2.c` `jsrc\verbs\vx.c`|xrem|jtxrem|`(^\|[ \t]+\|[^\d\w_])xrem\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define xtymes(x,y)                 jtxtymes(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\k.c` `jsrc\vq.h` `jsrc\vx.h` `jsrc\adverbs\ao.c` `jsrc\format\f2.c` `jsrc\verbs\v2.c` `jsrc\verbs\vq.c` `jsrc\verbs\vx.c`|xtymes|jtxtymes|`(^\|[ \t]+\|[^\d\w_])xtymes\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define zcir(x,y)                   jtzcir(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define zdiv(x,y)                   jtzdiv(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ap.c` `jsrc\verbs\v0.c` `jsrc\verbs\vbang.c` `jsrc\verbs\vgauss.c` `jsrc\verbs\vz.c` `jsrc\words\wn.c`|zdiv|jtzdiv|`(^\|[ \t]+\|[^\d\w_])zdiv\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define zeq(x,y)                    jtzeq(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vi.c`|zeq|jtzeq|`(^\|[ \t]+\|[^\d\w_])zeq\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define zgcd(x,y)                   jtzgcd(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vz.c`|zgcd|jtzgcd|`(^\|[ \t]+\|[^\d\w_])zgcd\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define zlcm(x,y)                   jtzlcm(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\ja.h`|#define zminus(x,y)                 jtzminus(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ap.c` `jsrc\verbs\v0.c` `jsrc\verbs\vbang.c` `jsrc\verbs\vgauss.c` `jsrc\verbs\vz.c`|zminus|jtzminus|`(^\|[ \t]+\|[^\d\w_])zminus\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define zplus(x,y)                  jtzplus(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ao.c` `jsrc\adverbs\ap.c` `jsrc\verbs\v0.c` `jsrc\verbs\vbang.c` `jsrc\verbs\vz.c` `jsrc\words\wn.c`|zplus|jtzplus|`(^\|[ \t]+\|[^\d\w_])zplus\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define zpow(x,y)                   jtzpow(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\v0.c` `jsrc\verbs\vbang.c` `jsrc\words\wn.c`|zpow|jtzpow|`(^\|[ \t]+\|[^\d\w_])zpow\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define zrem(x,y)                   jtzrem(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\verbs\vz.c`|zrem|jtzrem|`(^\|[ \t]+\|[^\d\w_])zrem\s*\((?=([^,]+?),([^)]+?)\))`||
|`jsrc\ja.h`|#define ztymes(x,y)                 jtztymes(jt,(x),(y))||`(#define\s+([\w\d]+)\(([^,]+),([^),]+)\)\s+(jt[\d\w]+)\(jt,\(\3\),\(\4\)\))`|``|
|`jsrc\adverbs\ao.c` `jsrc\adverbs\ap.c` `jsrc\verbs\v0.c` `jsrc\verbs\vbang.c` `jsrc\verbs\vgauss.c` `jsrc\verbs\vz.c` `jsrc\words\wn.c`|ztymes|jtztymes|`(^\|[ \t]+\|[^\d\w_])ztymes\s*\((?=([^,]+?),([^)]+?)\))`||

### CODE REVIEW

**Comment by Sebanisu** [comment]
The Automatic assign does not seem to work on draft pull requests.

**Review by codereport — APPROVED** [review]
:+1: will merge once conflicts resolved (sorry should have merged this one first)

**Comment by Sebanisu** [comment]
There weren't many conflicts. I think, I resolved them correctly. :)

**Pull request merged**
